ltp/data.py

```diff
@@ -90,28 +99,57 @@ def arguments(self):
         """
         return self._args
 
+    @property
+    def parallelizable(self):
+        """
+        If True, test can be run in parallel.
+        """
+        return self._parallelizable
+
 
-def read_runtest(suite_name: str, content: str) -> Suite:
+PARALLEL_BLACKLIST = [
+    "needs_root",
+    "needs_device",
+    "mount_device",
+    "mntpoint",
+    "resource_file",
+    "format_device",
+    "forks_child"
+]
+
+
+# pylint: disable=too-many-locals
+def read_runtest(
+        suite_name: str,
+        content: str,
+        metadata: dict = None) -> Suite:
     """
     It reads a runtest file content and it returns a Suite object.
     :param suite_name: name of the test suite
     :type suite_name: str
     :param content: content of the runtest file
     :type content: str
+    :param metadata: metadata JSON file content
+    :type metadata: dict
     :returns: Suite
     """
     if not content:
         raise ValueError("content is empty")
 
-    LOGGER.info("collecting testing suite: %s", suite_name)
+    LOGGER.info("Collecting testing suite: %s", suite_name)
```

Comment:
nit for next time: unrelated changes are better to be in separate commit (easier to review).